docs/rackspace-infrastructure-deployment.md

```diff
@@ -0,0 +1,83 @@
+## Rackspace Infrastructure Deployment 
+
+### Ironic for bare-metal provisioning 
+
+Rackspace TEchnology uses OpenStack bare metal provisioning, a.k.a **Ironic**, which provides bare metal machines instead of virtual machines, forked from the Nova baremetal driver. It is best thought of as a bare metal hypervisor API and a set of plugins which interact with the bare metal hypervisors. By default, it will use PXE and IPMI in order to provision and turn on/off machines, but Ironic also supports vendor-specific plugins which may implement additional functionality. 
+
+After switch and firewall configuration, deployment nodes are created with in the environment which host the required Ironic services:
+
+- PXE
+- DHCP
+- NBP
+- TFTP
+- IPMI
+
+### Ironic Diagram
+
+![conceptual_architecture](./assets/images/conceptual_architecture.png)
```

Comment:
We should omit heat and make it more agnostic to general IAC, infrastructure  as code such as terraform, ansible